test/e2e/feature/feature.go

```diff
@@ -202,6 +202,11 @@ var (
 	// TODO: document the feature (owning SIG, when to use this feature for a test)
 	MasterUpgrade = framework.WithFeature(framework.ValidFeatures.Add("MasterUpgrade"))
 
+	// Owner: sig-node
+	// Marks tests that require parallel image pull
+	// As the image pulling delay is needed, this feature needs a CRI proxy to inject the delay and this cannot be a general E2E.
+	MaxParallelImagePull = framework.WithFeature(framework.ValidFeatures.Add("MaxParallelImagePull"))
```

Comment:
Just use CRIProxy instead of introducing a new feature. The feature is in beta so it will not need any additional test job configuration

Reply:
Updated.